bocoup/aria-at-app
# Adding conflict handling

Ok, to test this is a bit obnoxious

1. Make another user in the database:
`insert into users(username) values ("foobar");`
`insert into user_to_at(user_id, at_name_id) values (FOOBAR_ID, AT_NAME_ID);`

2. Add yourself and the new user to a test run of a test cycle you initiated.

3. Save some test results for that test run AS YOURSELF

4. Update those test results so that they belong to 'foodbar' in the db

`update test_results set (user_id=FOOBAR_USER_ID);`

5. Now go save tests as yourself

6. To see the conflicts, you will have to finish a test then go back to it. I should make it so that they show up when you click "next test" but I ran out of steam for now!
